libraries/apollo-api/src/jvmMain/kotlin/com/apollographql/apollo3/api/Assertions.jvm.kt

```diff
@@ -0,0 +1,27 @@
+@file:JvmMultifileClass
+@file:JvmName("Assertions")
+
+package com.apollographql.apollo3.api
+
+@SafeVarargs
+@Suppress("NewApi")
```

Comment:
Is this safe?

Reply:
Actually no 😱. Good catch! `Optional` itself isn't a problem since this method will only be called in projects that have it - however `isEmpty()` is level 33+. I'll use `isPresent()` instead which is 24+, same as `Optional` itself.

Reply:
Add a comment in the code that it's safe to use `24+` because it's the same as `Optional` itself?